app/models/user.rb

```diff
@@ -0,0 +1,11 @@
+class User < ApplicationRecord
+  has_many :votes
+  has_many :works, through: :votes
+
```

Comment:
There is one goofy edge case not accounted for: validating on the presence of `name`!